libbeat/publisher/publish.go

```diff
@@ -87,6 +87,14 @@ type BeatPublisher struct {
 	numClients uint32
 }
 
+type BeatInfo struct {
```

Comment:
I'd like it to be `beat.Info`, not `common.BeatInfo`. With beats being kind of a framework, it would be nice if the `beat` (or `libbeat`) namespace would be the only packages libbeat and the actual beats would meat. Like: `beat.Client` being the instance used to publish a `beat.Event`. Only problem is initialization and circular dependencies due to having the main in `beat` package as well.